src/components/Image/Image.tsx

```diff
@@ -5,6 +5,21 @@ import { customPropTypes, UIComponent } from '../../lib'
 import { ImageBehavior } from '../../lib/accessibility'
 import { Accessibility } from '../../lib/accessibility/interfaces'
 
+import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
+import { Extendable, ReactChildren } from '../../../types/utils'
+
+export interface IImageProps {
+  accessibility?: Accessibility
+  as?: any
```

Comment:
Oops, I was adding the accessibility prop as object | Function, as I didn't know that there is prop for this. We may need to update this in the other components

Reply:
sure, will do